core/src/test/java/org/infinispan/distribution/RemoteGetFailureTest.java

```diff
@@ -0,0 +1,240 @@
+package org.infinispan.distribution;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.fail;
+
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Stream;
+
+import org.infinispan.Cache;
+import org.infinispan.commands.read.GetCacheEntryCommand;
+import org.infinispan.commands.remote.ClusteredGetCommand;
+import org.infinispan.commons.CacheException;
+import org.infinispan.commons.equivalence.AnyEquivalence;
+import org.infinispan.configuration.cache.CacheMode;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.context.InvocationContext;
+import org.infinispan.interceptors.DDAsyncInterceptor;
+import org.infinispan.manager.EmbeddedCacheManager;
+import org.infinispan.remoting.RemoteException;
+import org.infinispan.remoting.RpcException;
+import org.infinispan.remoting.inboundhandler.DeliverOrder;
+import org.infinispan.remoting.responses.CacheNotFoundResponse;
+import org.infinispan.remoting.responses.Response;
+import org.infinispan.remoting.responses.SuccessfulResponse;
+import org.infinispan.remoting.rpc.ResponseMode;
+import org.infinispan.remoting.rpc.RpcOptions;
+import org.infinispan.remoting.transport.jgroups.JGroupsTransport;
+import org.infinispan.test.MultipleCacheManagersTest;
+import org.infinispan.test.TestingUtil;
+import org.infinispan.test.fwk.CleanupAfterMethod;
+import org.infinispan.util.ByteString;
+import org.jgroups.Address;
+import org.jgroups.Channel;
+import org.jgroups.View;
+import org.jgroups.protocols.pbcast.GMS;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+
+@Test(groups = "functional")
+@CleanupAfterMethod
+public class RemoteGetFailureTest extends MultipleCacheManagersTest {
+   private Object key;
+
+   @Override
+   protected void createCacheManagers() throws Throwable {
+      ConfigurationBuilder builder = getDefaultClusteredCacheConfig(CacheMode.DIST_SYNC);
+      // cache stop takes quite long when the view splits
+      builder.clustering().stateTransfer().timeout(10, TimeUnit.SECONDS);
+      createCluster(builder, 3);
+      waitForClusterToForm();
+      key = getKeyForCache(cache(1), cache(2));
+   }
+
+   @AfterMethod(alwaysRun = true)
+   @Override
+   protected void clearContent() throws Throwable {
+      // do not clear the caches
+      TestingUtil.killCacheManagers(false, cacheManagers.toArray(new EmbeddedCacheManager[cacheManagers.size()]));
```

Comment:
Is there any reason in particular to avoid the clear? If so, please add a comment.